### PR TITLE
News customdata (has clicks) and news setorigin

### PR DESCRIPTION
**Pull request by therealfnadeau**

### CODE REVIEW

File: Scenarios/BTScenarios.json [hunk 1]
```diff
@@ -275,99 +278,119 @@
...
 		{
 			"name"   : "Scenarios deflected (Bravia)",
 			"weight" : 3,
 			"events" : [
-				{ "type" : "Search", "arguments" : { "queryText" : "Bravia", "goodQuery" : true, "caseSearch" : true, "inputTitle" : "Subject" } }
-			]
+				{ "type" : "SetOrigin", "arguments" : {"originLevel1" : "BesttechCommunityCaseCreation"}},
+				{ "type" : "Search", "arguments" : { "queryText" : "Bravia", "goodQuery" : true, "caseSearch" : true, "inputTitle" : "Subject" } },
+				{ "type" : "Custom", "arguments" : {"actionCause":"unloadPage", "actionType":"caseCreation", "customData" : { "hasClicks": false }} }
+<<<<<<< HEAD
```
Comment (erocheleau) [inline Scenarios/BTScenarios.json]: You need to remove these to resolve the conflicts.

File: Scenarios/BTScenarios.json [hunk 2]
```diff
@@ -275,99 +278,119 @@
...
 		},
+=======
+		]},
+>>>>>>> 5739464159abd23ede23b7bf4b125e06d551f46b
```
Comment (erocheleau) [inline Scenarios/BTScenarios.json]: I think the version below the ===== 
is what you want to keep.

File: Scenarios/BTScenarios.json [hunk 3]
```diff
@@ -275,99 +278,119 @@
...
 		},
 		{
 			"name"   : "Scenarios not deflected (bx6r6)",
 			"weight" : 7,
 			"events" : [
+				{ "type" : "SetOrigin", "arguments" : {"originLevel1" : "BesttechCommunityCaseCreation"}},
 				{ "type" : "Search", "arguments" : { "queryText" : "Television bx6r6", "goodQuery" : true, "caseSearch" : true, "inputTitle" : "Subject" } },
-				{ "type" : "Custom", "arguments" : { "actionCause" : "submitButton", "actionType" : "caseCreation" } }
-			]
+				{ "type" : "Custom", "arguments" : { "actionCause" : "submitButton", "actionType" : "caseCreation", "customData" : { "hasClicks": true }} }				]
```
Comment (erocheleau) [inline Scenarios/BTScenarios.json]: closing bracket on a separate line 
]

File: Scenarios/BTScenarios.json (hunk 1 above)
Comment (therealfnadeau) [inline Scenarios/BTScenarios.json]: It's already done and commited

File: Scenarios/BTScenarios.json (hunk 2 above)
Comment (therealfnadeau) [inline Scenarios/BTScenarios.json]: It's already done and commited

File: Scenarios/BTScenarios.json (hunk 1 above)
Comment (erocheleau) [inline Scenarios/BTScenarios.json]: No, if I see it it means it's still there.

**Comment by erocheleau** [comment]
Fix the issues and then I will merge. You are close!

File: Scenarios/BTScenarios.json (hunk 3 above)
Comment (therealfnadeau) [inline Scenarios/BTScenarios.json]: Do you want me to send a new commit to mouve the bracket I juste changet it Local

**Comment by therealfnadeau** [comment]
Do I need to close and repush a new pull request?

File: Scenarios/BTScenarios.json (hunk 2 above)
Comment (erocheleau) [inline Scenarios/BTScenarios.json]: Same as above

File: Scenarios/BTScenarios.json (hunk 3 above)
Comment (erocheleau) [inline Scenarios/BTScenarios.json]: Yes, that's how it works

**Comment by erocheleau** [comment]
No, commit the fixes, push and it will update the pull request

**Comment by erocheleau** [comment]
It's building but everything looks good I think.

**Pull request merged**